src/command-line/index.js

```diff
@@ -7,7 +7,7 @@ var fsextra = require("fs-extra");
 var path = require("path");
 var Helper = require("../helper");
 
-program.version(pkg.version, "-v, --version");
+program.version(Helper.getVersion(), "-v, --version");
```

Comment:
> What do you mean by npm version?

Sorry, I should have defined terminology I was using:

- What I call "npm version" is that human-friendly "1.5.0", "2.0.0-pre.6" string, whether it comes from a git tag or from the `package.json` file (let's call that simply a "version" from now on)
- What I call "commit hash" is... well, a (7-char) git commit hash :-)

> The entire point of this PR is to move from just providing commit SHAs to something with more face value.

But that's the issue here. There is no face value at all on an instance that runs a given version + N commits on top of it. We cannot inform the user about a version if a single commit has been added to it. What if that commit reverts all or part of a version (think issues added in pre.6)? What if that commit adds bugs? Should the users blame/report about that version or that commit?
For those reasons, I'd much rather take no "version responsibility" as soon as a single commit has been added to an instance.

> What about the version displayed on the client? Does this apply to the client as well?

You raise a good point and I believe versions should match in both the console and the UI.
If we want to push the 🚲 🏠 further, if not on a version, console could list the latest version + all commits added to that version, but we're going waaaaay too far! After all, we can't be doing the admin's job when it comes to using git 😅
So yeah, I definitely believe same on console + UI, and if admin wants more info, well, `git log` is there for that.